.github/workflows/deploy-pages.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright Avinal Kumar 2021
```

Comment:
Just a suggestion, it will be good to add your email to, e.g. `Copyright 2021 Avinal Kumar <185067@nith.ac.in>`